src/greyVest/Tag.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import F from 'futil'
+import _ from 'lodash/fp'
```

Comment:
this is your lint error

Reply:
vs code unused variable lowlighting doesn't really work on the underscore. fixed